src/redux/stores/transactions/wrapEventSubscriber.ts

```diff
@@ -0,0 +1,43 @@
+import { Web3Provider } from '@ethersproject/providers';
```

Comment:
imports linter error

Reply:
What's the error?

Reply:
Ah it was delayed for me. Just a sort warning.